kanshi.5.scd

```diff
@@ -35,6 +35,37 @@ Directives are followed by space-separated arguments. Arguments can be quoted
 	On *sway*(1), output names and descriptions can be obtained via
 	*swaymsg -t get_outputs*.
 
+*exec* <command>
+	An exec directive executes a command when the profile was successfully
+	applied. This can be used to update the compositor state to the profile
+	when not done automatically.
+
+	Commands are executed executed asynchronously and their order may not be
```

Comment:
typo in the doc: `executed` is written twice here

Reply:
Tahnks !